import ChainedError from "typescript-chained-error";
export declare class UniverseLogError extends ChainedError {
    constructor(message?: string, cause?: Error);
}
